### ISSUE TITLE
Remove commas from "desc," [BUG]

### ISSUE THREAD

**Issue by karikarshivani** [CMSgov/saf#57]
**Describe the bug**
Recommendations for the Notes section have a comma after "desc" which cause an error during execution

**To Reproduce**
1. Add the Notes text from Page 19 in the [Overlay SOP](https://cmsgov.github.io/saf/CMS_InSpec_Profile_Lifecycle_SOP_v1.0_20190702.pdf) to overlay.rb
2. Execute the overlay

**Expected behavior**
The overlay is expected to run smoothly

**Comment by aaronlippold** [comment]
sub-descriptions should have a comma

**Comment by aaronlippold** [comment]
If you look at the windows-10 profile for example:

desc "check", "If the following registry value does not exist or is not"


lippold@gmail.com

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'

**Comment by karikarshivani** [comment]
@aaronlippold, yes, however after the keywords. The example listed in the "To Reproduce" section in my first comment has the following:
```
desc, 'caveat', 'Not applicable for this CMS ARS 3.1 overlay, since the related security control is not included in
CMS ARS 3.1'
```

The `desc, 'caveat',` needs to be changed to `desc 'caveat',`

**Comment by karikarshivani** [comment]
Screenshot from the Overlay SOP for reference:
![image](https://user-images.githubusercontent.com/11844975/90073111-a3534480-dcc6-11ea-89bb-95df8868c253.png)
